src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java

```diff
@@ -235,11 +236,12 @@ public void onBindHeaderViewHolder(HeaderViewHolder viewHolder, int position) {
         txt = context.getString(R.string.profile_shared_chats);
         break;
       case ItemData.TYPE_PRIMARY_SETTING:
-        if(itemDataContact!=null && itemDataContact.isVerified()) {
-          txt = context.getString(R.string.verified_contact);
```

Comment:
So, we don't show whether a contact is verified anymore?

I think it should somehow be visible in the profile that a contact is verified

Reply:
it is shown with the blue verified-checkbox, this is what we are doing on desktop/ios as well, so, probably fine.

Reply:
I couldn't find the blue verified-checkbox anywhere, but maybe that's the actual problem

Reply:
it is right of the names on android.

ios and desktop shows them on avatars, which is maybe better. however, as this "verified" stuff seems not to be totally settled yet, i am not suggesting to change things here on android :)

Reply:
yip that is what does this PR now the verified checkbox is added to the name, currently it is not show in the chat list, should I also display it there? it is a bit confusing with verified groups but IMHO "verified groups" should be rebranded to "protected group" and use a shield icon for protected chats and let the verified icon for verified contacts then display verified icon everywhere for verified contacts

Reply:
All of this is out of scope for this PR, so consider this approved :)

But at least on my device there is no checkmark shown next to the name
![image_2021-11-23_17-10-17](https://user-images.githubusercontent.com/18012815/143071554-941f5e35-8e4c-4833-a506-477bbfde8b01.png)

Reply:
hocuri, check this PR's commits, the checkmark will be shown with this PR it was not show in the DC version you are using that is what this PR changes, it removes the "verified contact" header and instead shows a checkmark